src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -75,37 +87,56 @@ ActorTemplates:
             ThrowOnFailure: false
 Actors:
 - Name: Setup
-  Type: Loader
+  Type: AdminCommand
   Threads: 1
   Phases:
     OnlyActiveInPhases:
       Active: [0]
       NopInPhasesUpTo: *NumPhases
       PhaseConfig:
-        # Have 500 documents ready at start
         Repeat: 1
-        BatchSize: 100
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            ttlMonitorSleepSecs: 1
+        SleepAfter: 60 seconds  # Wait 60 seconds so that the new ttl sleep config is active.
```

Comment:
Do we still need this even after [SERVER-70055](https://jira.mongodb.org/browse/SERVER-70055)? I guess we'll still need it for old branches if we're running this workload there?

Reply:
Yea, I think so. Since this is just the setup phase, it doesn't add a lot of overhead to the workload.